pages/vote/elo.tsx

```diff
@@ -72,6 +88,12 @@ export default function Elo({ sessions }: EloProps): JSX.Element {
 
   async function sessionChoiceHandler(winningSession: EloSession, losingSession: EloSession, isDraw = false) {
     await postPair(winningSession.Id, losingSession.Id, isDraw)
+    logEvent('voting', 'vote', {
+      variant: layoutVariant,
+      winningVote: winningSession.Id,
+      losingSession: losingSession.Id,
+      isDraw,
```

Comment:
I'm not sure if this is enough to tell us whether they chose option 1 or option 2 on its own.

We could add details from the `sessionPair`
```diff
logEvent('voting', 'vote', {
  variant: layoutVariant,
+ sessionA: sessionPair.SubmissionA.Id,
+ sessionB: sessionPair.SubmissionB.Id,
  winningVote: winningSession.Id,
  losingSession: losingSession.Id,
  isDraw,
})
```

Reply:
That makes sense. Have added the extra information